src/RoomInvite.tsx

```diff
@@ -44,15 +44,18 @@ export interface IInviteResult {
  * @param {string} roomId The ID of the room to invite to
  * @param {string[]} addresses Array of strings of addresses to invite. May be matrix IDs or 3pids.
  * @param {function} progressCallback optional callback, fired after each invite.
+ * @param {boolean} sendSharedHistoryKeys whether to share e2ee keys with the invitees if applicable.
  * @returns {Promise} Promise
  */
 export function inviteMultipleToRoom(
     roomId: string,
     addresses: string[],
     progressCallback?: () => void,
+    sendSharedHistoryKeys = false,
```

Comment:
Maybe better before the callback arg...? Or in an options object, so it's named at the call site? Just suggestions though, no strong opinions either way.